build/props/common.props

```diff
@@ -38,4 +38,8 @@
     <ItemGroup Condition="'$(SilkEnableSourceLink)' != '' And '$(SilkNativePackage)' == ''">
         <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All"/>
     </ItemGroup>
+    <PropertyGroup Condition="'$(SilkNativePackage)' == 'true'">
```

Comment:
nit: either we use `'$(A)' != ''` as "A is set" or we use `$(A)' == 'true'`

Reply:
Noted
```suggestion
    <PropertyGroup Condition="'$(SilkNativePackage)' != ''">
```